packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -82,8 +82,8 @@ export class Interpreter {
 		this.runtime = <Runtime>{};
 		this.callStack = new Stack<number>();
 		this.labelMap = new Map<string, number>();
-		this.subTxn = undefined;
-		this.innerTxns = [];
+		this.currentInnerTxGroup = [];
+		this.innerTxnGroups = [];
```

Comment:
What is the difference between `Tx` and `Txn`?

Reply:
It's same

Reply:
Maybe we could just use one?

Reply:
updated